frontend/src/app/component/phenotypic-feature/phenotypic-feature.component.html

```diff
@@ -0,0 +1,84 @@
+                        <h3>{{termId}}</h3>
+                    </div>
+                </div>
+                <div class="row">
+                    <p>
+                        {{description}}
+                    </p>
+                </div>
+
+                <div class="row">
+                    <div class="col-md-12">
+                        <label id="status-radio-group-label"><b>Status:</b></label>
+                    </div>
+                    <div class="col-md-12">
+                        <mat-radio-group aria-labelledby="status-radio-group-label" class="status-radio-group"
+                            [(ngModel)]="selectedStatus">
+                            <mat-radio-button class="status-radio-button" *ngFor="let status of statuses"
+                                [value]="status">
+                                {{status}}
+                            </mat-radio-button>
+                        </mat-radio-group>
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="col-md-12">
+                        <label id="onset-time-label"><b>Onset:</b></label>
+                    </div>
+                    <div class="col-md-12">
+                        <app-age-control aria-labelledby="onset-time-label"></app-age-control>
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="col-md-12">
+                        <label id="resolution-time-label"><b>Resolution:</b></label>
+                    </div>
+                    <div class="col-md-12">
+                        <app-age-control aria-labelledby="resolution-time-label"></app-age-control>
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="col-md-12">
+                        <label id="severity-label"><b>Severity:</b></label>
+                    </div>
+                    <div class="col-md-12">
+                        <app-search-filter [options]="severities" title="Add severity"></app-search-filter>
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="col-md-12">
+                        <label id="modifiers-label"><b>Modifiers:</b></label>
+                    </div>
+                    <div class="col-md-12">
+                        <app-select-chip [values]="modifierValues" label="Modifier values"
+                            placeholderLabel="Add Modifier Value..." description="A description of the modifier UI">
+                        </app-select-chip>
+                    </div>
+                </div>
+
+                <div class="row">
+                    <div class="col-md-12">
+                        <label id="evidence-label"><b>Evidence:</b></label>
+                    </div>
+                    <div class="col-md-12">
```

Comment:
I think I made an error in the Figma specification. The evidence is not a simple constant/value, but it is a data structure that has several attributes as well. 

This element (list of the present evidence) should summarize the present items. A new element should be added in a separate widget that accepts *Evidence code*, *ID*, *Reference*, and *Description*.

I think this would be easier to discuss in a call.